plugins/vendor/lib/model/VendorIntegrationPeer.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+
+/**
+ * Skeleton subclass for performing query and update operations on the 'vendor_integration' table.
+ *
+ * 
+ *
+ * You should add additional methods to this class to meet the
+ * application requirements.  This class will only be generated as
+ * long as it does not already exist in the output directory.
+ *
+ * @package plugins.vendor
+ * @subpackage model
+ */
+class VendorIntegrationPeer extends BaseVendorIntegrationPeer {
+
```

Comment:
I am missing the setDefaultCriteriaFilter method which will filter out any objects that are in statuses which you do not want to get back in the result.

Reply:
generated automatically via propel